apps/coscrad-frontend/src/components/tree-of-knowledge/category-tree-ui-context.tsx

```diff
@@ -0,0 +1,17 @@
+import { createContext, useState } from 'react';
+import {
+    CategoryTreeNodeDepth,
+    CategoryTreeUXContextType,
+} from './types/category-tree-ui-context-type';
+
+export const CategoryTreeUXContext = createContext<CategoryTreeUXContextType | null>(null);
+
+export const CategoryTreeUXProvider = ({ children }): JSX.Element => {
```

Comment:
Is this really necessary? I'd use context sparingly.

Reply:
Removed Context